modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -1076,48 +948,54 @@ void RenderableFieldlinesSequence::render(const RenderData& data, RendererTasks&
-        _shaderProgram->setUniform("usingDomain",  _pDomainEnabled);
-        _shaderProgram->setUniform("usingMasking", _pMaskingEnabled);
+        _shaderProgram->setUniform("colorMethod",  _colorMethod);
+        _shaderProgram->setUniform("lineColor",    _colorUniform);
+        _shaderProgram->setUniform("usingDomain",  _domainEnabled);
+        _shaderProgram->setUniform("usingMasking", _maskingEnabled);
 
-        if (_pColorMethod == static_cast<int>(ColorMethod::ByQuantity)) {
+        if (_colorMethod == static_cast<int>(ColorMethod::ByQuantity)) {
             ghoul::opengl::TextureUnit textureUnit;
             textureUnit.activate();
             _transferFunction->bind(); // Calls update internally
             _shaderProgram->setUniform("colorTable", textureUnit);
             _shaderProgram->setUniform("colorTableRange",
-                                            _colorTableRanges[_pColorQuantity]);
+                                            _colorTableRanges[_colorQuantity]);
         }
 
-        if (_pMaskingEnabled) {
-            _shaderProgram->setUniform("maskingRange", _maskingRanges[_pMaskingQuantity]);
+        if (_maskingEnabled) {
+            _shaderProgram->setUniform("maskingRange", _maskingRanges[_maskingQuantity]);
         }
 
-        _shaderProgram->setUniform("domainLimR", _pDomainR.value() * _scalingFactor);
-        _shaderProgram->setUniform("domainLimX", _pDomainX.value() * _scalingFactor);
-        _shaderProgram->setUniform("domainLimY", _pDomainY.value() * _scalingFactor);
-        _shaderProgram->setUniform("domainLimZ", _pDomainZ.value() * _scalingFactor);
+        _shaderProgram->setUniform("domainLimR", _domainR.value() * _scalingFactor);
+        _shaderProgram->setUniform("domainLimX", _domainX.value() * _scalingFactor);
+        _shaderProgram->setUniform("domainLimY", _domainY.value() * _scalingFactor);
+        _shaderProgram->setUniform("domainLimZ", _domainZ.value() * _scalingFactor);
 
         // Flow/Particles
-        _shaderProgram->setUniform("flowColor",       _pFlowColor);
-        _shaderProgram->setUniform("usingParticles",  _pFlowEnabled);
-        _shaderProgram->setUniform("particleSize",    _pFlowParticleSize);
-        _shaderProgram->setUniform("particleSpacing", _pFlowParticleSpacing);
-        _shaderProgram->setUniform("particleSpeed",   _pFlowSpeed);
+        _shaderProgram->setUniform("flowColor",       _flowColor);
+        _shaderProgram->setUniform("usingParticles",  _flowEnabled);
+        _shaderProgram->setUniform("particleSize",    _flowParticleSize);
+        _shaderProgram->setUniform("particleSpacing", _flowParticleSpacing);
+        _shaderProgram->setUniform("particleSpeed",   _flowSpeed);
         _shaderProgram->setUniform(
             "time",
-            global::windowDelegate->applicationTime() * (_pFlowReversed ? -1 : 1)
+            global::windowDelegate->applicationTime() * (_flowReversed ? -1 : 1)
```

Comment:
Are you sure this is correct?  The application time shouldn't be inverted, shouldn't it be the delta time instead?

Reply:
I havent touched this before so i am not sure. I don't know exactly what the applicationTime is, but since deltaTime is more clear ill use that.